### PR TITLE
Dev kelly

### PR DESCRIPTION
**Pull request by omnivector-kelly**
Add support for Juju Resources for NHC.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,29 +12,32 @@ clean: ## Remove all deployed applications
 ${ETCD}: ## Download etcd resource
 	wget https://github.com/etcd-io/etcd/releases/download/v3.5.0/${ETCD}
 
+${NHC}:
```
Comment (heitorPB) [inline Makefile:15]:
```suggestion
${NHC}: ## Download NHC resource
```

File: Makefile [hunk 2]
```diff
@@ -11,29 +12,32 @@ clean: ## Remove all deployed applications
...
+	wget https://github.com/mej/nhc/releases/download/1.4.3/${NHC}
+
 .PHONY: lxd-focal
-lxd-focal: ${ETCD} ## Deploy slurm-core in a local LXD Ubuntu Focal cluster
+lxd-focal: ${ETCD} ${NHC}## Deploy slurm-core in a local LXD Ubuntu Focal cluster
```
Comment (heitorPB) [inline Makefile:19]:
```suggestion
lxd-focal: ${ETCD} ${NHC} ## Deploy slurm-core in a local LXD Ubuntu Focal cluster
```

File: Makefile [hunk 3]
```diff
@@ -11,29 +12,32 @@ clean: ## Remove all deployed applications
...
 	juju deploy ./slurm-core/bundle.yaml \
 	            --overlay ./slurm-core/clouds/lxd.yaml \
 	            --overlay ./slurm-core/series/focal.yaml \
 	            --overlay ./slurm-core/charms/local-development.yaml
 
 .PHONY: lxd-centos
-lxd-centos: ${ETCD} ## Deploy slurm-core in a local LXD CentOS7 cluster
+lxd-centos: ${ETCD} ${NHC}## Deploy slurm-core in a local LXD CentOS7 cluster
```
Comment (heitorPB) [inline Makefile:26]:
```suggestion
lxd-centos: ${ETCD} ${NHC} ## Deploy slurm-core in a local LXD CentOS7 cluster
```

File: Makefile [hunk 4]
```diff
@@ -11,29 +12,32 @@ clean: ## Remove all deployed applications
...
 	juju deploy ./slurm-core/bundle.yaml \
 	            --overlay ./slurm-core/clouds/lxd.yaml \
 	            --overlay ./slurm-core/series/centos7.yaml \
 	            --overlay ./slurm-core/charms/local-development.yaml
 
 .PHONY: aws-focal
-aws-focal: ${ETCD} ## Deploy slurm-core in a AWS Ubuntu Focal cluster
+aws-focal: ${ETCD} ${NHC}## Deploy slurm-core in a AWS Ubuntu Focal cluster
```
Comment (heitorPB) [inline Makefile:33]:
```suggestion
aws-focal: ${ETCD} ${NHC} ## Deploy slurm-core in a AWS Ubuntu Focal cluster
```

File: Makefile [hunk 5]
```diff
@@ -11,29 +12,32 @@ clean: ## Remove all deployed applications
...
 	juju deploy ./slurm-core/bundle.yaml \
 	            --overlay ./slurm-core/clouds/aws.yaml \
 	            --overlay ./slurm-core/series/focal.yaml \
 	            --overlay ./slurm-core/charms/local-development.yaml
 
 .PHONY: aws-centos
-aws-centos: ${ETCD} ## Deploy slurm-core in a AWS CentOS7 cluster
+aws-centos: ${ETCD} ${NHC}## Deploy slurm-core in a AWS CentOS7 cluster
```
Comment (heitorPB) [inline Makefile:40]:
```suggestion
aws-centos: ${ETCD} ${NHC} ## Deploy slurm-core in a AWS CentOS7 cluster
```

File: slurm-core/bundle.yaml [hunk 6]
```diff
@@ -14,6 +14,8 @@ applications:
   slurmd:
     charm: slurmd
     num_units: 1
+    resources:
+      nhc: ./../lbnl-nhc-1.4.3.tar.gz
```
Comment (heitorPB) [inline slurm-core/bundle.yaml:18]: Could you move this to the `slurm-core/charms/local-development.yaml` file? We only need to pass a path to a local file in our dev setup, when not a dev setup we can get the resource from Charmhub.

**Review by heitorPB — CHANGES_REQUESTED** [review]
Nice work! I have left some minor suggestions for your PR.

File: slurm-core/bundle.yaml (hunk 6 above)
Comment (omnivector-kelly) [reply slurm-core/bundle.yaml]: This has been moved. https://github.com/omnivector-solutions/slurm-bundles/commit/2e67d6e0bd49038f15fe08470441506d253c5daf

**Comment by heitorPB** [comment]
@omnivector-kelly could you rebase the code to solve the conflicts please?

**Review by heitorPB — APPROVED** [review]
Nice work!

**Pull request merged**